rust/chains/hyperlane-cardano/src/bin/cardano_rpc_main.rs

```diff
@@ -0,0 +1,59 @@
+use hyperlane_cardano::rpc::CardanoRpc;
+use hyperlane_cardano::{
+    CardanoMailbox, CardanoMailboxIndexer, CardanoValidatorAnnounce, ConnectionConf,
+};
+use hyperlane_core::{
+    ChainResult, ContractLocator, HyperlaneDomain, HyperlaneMessage, IndexRange, Indexer,
+    KnownHyperlaneDomain, Mailbox, ValidatorAnnounce, H256,
+};
+use std::str::FromStr;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() -> ChainResult<()> {
+    let cardano_rpc = CardanoRpc::new(&"http://localhost:3000".parse().unwrap());
```

Comment:
Can we pass this in as an environment variable?

Reply:
This seems fine as it looks like a test binary used only for development, not code bundled in the agent binaries.

Reply:
This file is only used for tests. Full deployment uses RPC URL provided from configs.